src/tree/aksClusterTreeItem.ts

```diff
@@ -36,7 +36,11 @@ export default class AksClusterTreeItem extends AzureTreeItem implements AksClus
     }
 
     public get name(): string {
-        return this.resource.name || '';
+        return this.resource.name!;
```

Comment:
Note: I changed this because that's my preferred approach for the `resourceType` property I'm adding, and I think they should be consistent.

If both `name` and `type` on the resource really should always be defined (in this context at least), then I think it makes sense for the code to throw rather than return empty string (which would fail in weird ways in other places).

On the other hand, if they might not always be defined, then we should change these properties to return `string | undefined` and let consumers deal with it.